src/eva/data/ioda_obs_space.py

```diff
@@ -98,7 +98,9 @@ def execute(self, data_collections):
                 ds_groups = xr.Dataset()
 
                 # Save sensor_channels for later
-                sensor_channels = ds_header['nchans']
+sensor_channels = 0
```

Comment:
@ADCollard great indentation! :-)

Reply:
@CoryMartin-NOAA Not sure what happened there, I accepted some suggested changes and it ended up like that.